.gitignore

```diff
@@ -366,4 +366,5 @@ src/Nethermind/Nethermind.BeaconNode.Host/configuration/Output
 src/Nethermind/Nethermind.BeaconNode.Host/release
 
 # Memory dumps
-mono_crash.mem.*.blob
\ No newline at end of file
+mono_crash.mem.*.blob
```

Comment:
hmmm, what is this change about?

Reply:
dunno why it shows that I added this, this is in .gitignore right now

Reply:
you probably want to remove this file from commit then